docs/contributing/triaging.md

```diff
@@ -0,0 +1,141 @@
+# Triaging guide
+
+With constant influx of new issues, it's essential to prioritize and categorize them efficiently to ensure that the most important problems are addressed promptly. This document outlines our approach to triaging issues on GitHub, including guidelines for labeling and resolving issues, as well as best practices for maintaining a well-organized and up-to-date issue tracker.
+
+## Labeling Issues
+
+When labeling an issue, choose the label(s) that best describes the issue. Using labels consistently and accurately ensures that issues are trackable and searchable.
+
+
+### Issue Types
+
+- **type: bug**: Use for issues that identify bugs causing incorrect or unexpected behavior.
+- **type: duplicate**: Use this label for issues that are duplicates of existing ones.
+- **type: enhancement**: Use this label for requests for new features or improvements to existing functionalities.
+- **type: will not do**: Use this label for issues that won’t be addressed or fixed.
+
+### Impact labels
+
+These labels describe what portion of the project they affect:
+
+- **impact: breaking**: Issues that break our public API.
+- **impact: core**: Issues that affect the core Python codebase.
+- **impact: dependencies**: Use this label for issues concerning dependencies.
+- **impact: documentation**: Use this label for issues related to documentation.
+- **impact: automation**: Use this label for issues related to the CI/CD pipeline or automation
+
+### Needs labels
+
+- **needs: decision**: We're struggling to decide what to do and the decision committee needs to help.
+- **needs: feedback**:  Use this label for issues where feedback is requested from the team or our community.
+- **needs: help**: Use this label for issues where additional help or contributions are needed.
+
```

Comment:
I'd like to add back the "good first issue" label, perhaps alone in a "Special labels" section? That is one that triagers should be especially aware of!